checkov/common/runners/runner_registry.py

```diff
@@ -85,8 +85,8 @@ def print_reports(self, scan_reports, config, url=None):
                 print(json.dumps(report_jsons[0], indent=4))
             else:
                 print(json.dumps(report_jsons, indent=4))
-        if config.output == "cli":
-            bc_integration.get_report_to_platform(config,scan_reports)
+        #if config.output == "cli":
```

Comment:
@eurogig On purpose? If so - delete it, don't leave it commented out

Reply:
Ok will do